compiler/rustc_target/src/spec/mod.rs

```diff
@@ -504,6 +504,7 @@ supported_targets! {
     ("armv4t-unknown-linux-gnueabi", armv4t_unknown_linux_gnueabi),
     ("armv5te-unknown-linux-gnueabi", armv5te_unknown_linux_gnueabi),
     ("armv5te-unknown-linux-musleabi", armv5te_unknown_linux_musleabi),
+    ("armv5te-unknown-linux-uclibcgnueabi", armv5te_unknown_linux_uclibcgnueabi),
```

Comment:
```suggestion
    ("armv5te-unknown-linux-uclibcgnueabi", armv5te_unknown_linux_uclibceabi),
```
The naming seems inconsistent with other `armv5te-unknown-linux-*` and `*eabi* `targets.

Reply:
I understand. My original suggestion came from gcc, where they usually name this ABI as `uclibcgnueabi`. But I agree that it is inconsistent with the other targets. I'll rename it.